package.json

```diff
@@ -51,12 +54,13 @@
     "trailingComma": "es5"
   },
   "scripts": {
-    "start": "tsdx watch",
-    "build": "tsdx build",
+    "start": "npm run build:esm -- --watch",
+    "build": "rimraf dist && run-p build:*",
+    "build:esm": "tsc -p tsconfig.build.json --outDir dist/esm",
```

Comment:
Consuming TS projects will automatically pick the cjs `main` export, not the esm `module` one, but for future-proving and other consumers I've kept the multi-export.